pipeline/eval/requirements/eval.txt

```diff
@@ -1,221 +1,157 @@
 #
-# This file is autogenerated by pip-compile with Python 3.11
+# This file is autogenerated by pip-compile with Python 3.10
 # by the following command:
 #
-#    pip-compile --generate-hashes eval-requirements.in
+#    pip-compile pipeline/eval/requirements/eval.in
 #
-colorama==0.4.6 \
-    --hash=sha256:08695f5cb7ed6e0531a20572697297273c47b8cae5a63ffc6d6ed5c201be6e44 \
-    --hash=sha256:4f1d9991f5acc0ca119f9d443620b77f9d6b33703e51011c16baf57afb285fc6
+aiohttp==3.9.5
```

Comment:
It's difficult to argue that there's any real security risks to not pinning hashes, but could you please regenerate this with `--generate-hashes` anyways, to be safe? (I suppose the worst thing that could happen is pulling in cryptomining code or some such...)

Reply:
This wasn't an intentional change on my part. To guard against this in the future, I added a task to update the requirements.

https://github.com/mozilla/firefox-translations-training/pull/587/commits/27d130afd43f4c696eb80fa860517703ffb27a93